src/parse-mockdb.js

```diff
@@ -603,6 +657,11 @@ function objectsAreEqual(obj1, obj2) {
     return true;
   }
 
+  // relation
+  if (obj1.__type === 'Relation' && obj1.ids !== undefined && obj1.ids.indexOf(obj2.id || obj2.objectId) > -1) {
```

Comment:
Can we helper methods like `isRelation(object)` which tells me if an object is a relation, `getId(object)` which retrieves an id whether it is in the "id" key or the "objectId" key, and `relationHasId(relation, id)` for this?